languages/Cobol.groovy

```diff
@@ -339,3 +339,18 @@ def getRepositoryClient() {
 
 	return repositoryClient
 }
+
+boolean buildListContainsTests(List<String> buildList) {
+	boolean testFound = false
+	// iterate through build list
+	buildList.each { buildFile ->
+	
+		// Check if this is a testcase
+		isZUnitTestCase = (props.getFileProperty('cobol_testcase', buildFile).equals('true')) ? true : false
+		
+		if (isZUnitTestCase) {
+			testFound = true
```

Comment:
We can directly `return true` if the first test is found, instead of integrating through the entire buildList.

Reply:
Good idea, this will improve performance.